catalogs/catalog-lakehouse-paimon/src/test/java/org/apache/gravitino/catalog/lakehouse/paimon/integration/test/CatalogPaimonFileSystemIT.java

```diff
@@ -48,4 +55,22 @@ protected Map<String, String> initPaimonCatalogProperties() {
 
     return catalogProperties;
   }
+
+  @Test
+  void testPaimonSchemaProperties() throws Catalog.DatabaseNotExistException {
```

Comment:
no need to add this test for schema properties, since it's empty.

Reply:
removed.